lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs

```diff
@@ -244,101 +252,153 @@ spec = do
+    postExternalTxTest ctx headers payload = do
+        let toSend = 1 :: Natural
+        (ExternalTxFixture wSrc wDest fee txWits) <-
+                fixtureExternalTx ctx toSend
+        let encodedSignedTx = encodeTx txWits MsgTypeTransaction
+        r <-
+            request @ApiTxId ctx postExternalTxEp headers (payload encodedSignedTx)
+        verify r
             [ expectSuccess
             , expectResponseCode HTTP.status202
             ]
-        r2 <- request @ApiWallet ctx (getWalletEp wDest) Default Empty
-        verify r2
+
+        rb <- request @ApiWallet ctx (getWalletEp wDest) Default Empty
+        verify rb
             [ expectSuccess
-            , expectEventually ctx balanceAvailable amt
+            , expectEventually ctx balanceAvailable toSend
+            ]
+        ra <- request @ApiWallet ctx (getWalletEp wSrc) Default Empty
+        verify ra
+            [ expectEventually ctx balanceAvailable (faucetAmt - fee - toSend)
             ]
 
-        --at this point wDest wallet should have 100 in one of its address
-        --now we try to prepare (tx, [txWintess]), encode it and send 1 to
-        --empty wallet but using external-transactions endpoint
-        txsSrc <- listAllTransactions ctx wSrc
-        let txSrc:_ = filter (\tx ->
-                                any (\(AddressAmount _ amnt) ->
-                                         amnt == Quantity amt)
-                                $ NE.toList $ tx ^. #outputs
-                           ) txsSrc
-        let txHash = getApiT $ txSrc ^. #id
-        let out1@(AddressAmount _ amnt1):out2:_ =
-                NE.toList $ txSrc ^. #outputs
-        let (ix, out) = if amnt1 == Quantity amt then
-                (0::Word32, out1)
-                else
-                (1::Word32, out2)
-        let outAddr = getApiT $ fst $ API.address out
-        let myRnps = [(TxIn txHash ix, TxOut outAddr (Coin 100))]
-        wEmpt <- emptyWallet ctx
-        addrsEmpt <- listAddresses ctx wEmpt
-        let destEmpt = (addrsEmpt !! 1) ^. #id
-        let destAddr = getApiT $ fst destEmpt
-        let left = if amt >= (fromIntegral feeMin) + toSend then
-                fromIntegral $ amt - (fromIntegral feeMin) - toSend :: Word64
-                   else
-                0 :: Word64
-        let txOuts = [ TxOut addrD (Coin left)
-                     , TxOut destAddr (Coin (fromIntegral toSend))]
-        let inps = fmap (second coin) myRnps
-        let block0H = unsafeFromHex
-                "301b1c634aa7b586da7243dd66a61bde904bc1755e9a20a9b5b1b0064e70d904"
-        let bs = block0H <> getHash (signData inps txOuts)
-        let (Just result) = isOwned st' (rootXPrv, pwd) addrD
-        let wits = [sign bs result]
-        return (Tx (fragmentId inps txOuts wits) inps txOuts, wits)
+        let wronglyEncodedTx = encodeWronglyTx txWits MsgTypeTransaction
+        r1 <- request @ApiTxId ctx postExternalTxEp headers (payload wronglyEncodedTx)
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Reply:
corrected